src/Context/Context.js

```diff
@@ -6,9 +6,10 @@ import Swal from 'sweetalert2';
 import 'react-toastify/dist/ReactToastify.css';
 export const ContextData = React.createContext();
 function ContextProvider({ children }) {
+   
```

Comment:
suggest:
useless lines can make you confused.
before you push, you can remove.